kadai-adapter-camunda-listener/src/main/java/io/kadai/adapter/camunda/schemacreator/KadaiOutboxSchemaCreator.java

```diff
@@ -47,12 +47,12 @@ public class KadaiOutboxSchemaCreator {
       SQL + "/postgres/kadai-outbox-schema-postgres.sql";
   private static final String DB_SCHEMA_ORACLE = SQL + "/oracle/kadai-outbox-schema-oracle.sql";
 
-  private DataSource dataSource;
-  private String schemaName;
-  private StringWriter outWriter = new StringWriter();
-  private PrintWriter logWriter = new PrintWriter(outWriter);
-  private StringWriter errorWriter = new StringWriter();
-  private PrintWriter errorLogWriter = new PrintWriter(errorWriter);
+  private final DataSource dataSource;
```

Comment:
No, in general it is better to have finals, because it guarantees that the variable won't be reassigned. It could avoid some nasty bugs in complicated methods. I don't remember if SonarCloud or IntelliJ highlights it as an issue